hexapod_dart/src/hexapod_dart_simu.cpp

```diff
@@ -334,3 +291,86 @@ void HexapodDARTSimu::_add_floor()
 
     _world->addSkeleton(floor);
 }
+
+double HexapodDARTSimu::_min_dist_angle(double a1, double a2)
```

Comment:
Since you already have loops to ensure that the angle is between -180° and 180°, what's the point of the first part then ?